introducing-java/src/main/java/com/onto/javacourse/intro/summingLargeNumbers/SummingLargeNumbers.java

```diff
@@ -0,0 +1,44 @@
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This class provides a method to sum two large numbers represented as strings.
+ */
+public class SummingLargeNumbers {
+
+    /**
+     * Sums two large numbers represented as strings.
+     *
+     * @param fNum the first large number
+     * @param sNum the second large number
+     * @return a list representing the sum of the two large numbers
+     */
+    public static List<Integer> sumNumbers(String fNum, String sNum) {
+        if (fNum.length() < sNum.length()) {
+            String temp = sNum;
+            sNum = fNum;
+            fNum = temp;
+        }
+
+        List<Integer> sum = new ArrayList<>();
```

Comment:
I am a bit confused on why do you use an array for the result. To me it looks most logical to have the result the same way you have the input parameters - a string. Can you elaborate on why an array? Even if you want to use array internally, it would still be a good idea to change the method signature.